modules/ROOT/pages/errors/index.adoc

```diff
@@ -1,91 +1,208 @@
-// | 22G11	        | error: data exception - reference value, referent deleted
-// | 22G13	        | error: data exception - invalid group variable value
-// | 22G14	        | error: data exception - incompatible temporal instant unit groups
-// | 25000	        | error: invalid transaction state
-// | 25G01	        | error: invalid transaction state - active GQL-transaction
-// | 25G02	        | error: invalid transaction state - catalog and data statement mixing not supported
-// | 25G03	        | error: invalid transaction state - read-only GQL-transaction
-// | 25G04	        | error: invalid transaction state - accessing multiple graphs not supported
-// | 2D000	        | error: invalid transaction termination
-// | 40000	        | error: transaction rollback
-// | 40003	        | error: transaction rollback - statement completion unknown
-// | 42000	        | error: syntax error or access rule violation
-// | 42001	        | error: syntax error or access rule violation - invalid syntax
-// | 42002	        | error: syntax error or access rule violation - invalid reference
-// | 42004	        | error: syntax error or access rule violation - use of visually confusable identifiers
-// | 42006	        | error: syntax error or access rule violation - number of edge labels below supported minimum
-// | 42007	        | error: syntax error or access rule violation - number of edge labels exceeds supported maximum
-// | 42008	        | error: syntax error or access rule violation - number of edge properties exceeds supported maximum
-// | 42009	        | error: syntax error or access rule violation - number of node labels below supported minimum
-// | 42010	        | error: syntax error or access rule violation - number of node labels exceeds supported maximum
-// | 42011	        | error: syntax error or access rule violation - number of node properties exceeds supported maximum
-// | 42012	        | error: syntax error or access rule violation - number of node type key labels below supported minimum
-// | 42013	        | error: syntax error or access rule violation - number of node type key labels exceeds supported maximum
-// | 42014	        | error: syntax error or access rule violation - number of edge type key labels below supported minimum
-// | 42015	        | error: syntax error or access rule violation - number of edge type key labels exceeds supported maximum
-// | G1000	        | error: dependent object error
-// | G1001	        | error: dependent object error - edges still exist
-// | G1002	        | error: dependent object error - endpoint node is deleted
-// | G1003	        | error: dependent object error - endpoint node not in current working graph
-// | G2000	        | error: graph type violation
-// |===
-
-// [[gqlstatus-neo4j-defined-codes]]
-// === Neo4j-defined GQLSTATUS error codes
+The Neo4j DBMS returns a server error to indicate that the outcome of a Cypher query or command execution is unsuccessful.
```

Comment:
What is here "Neo4j DBMS" is elsewhere in the text "server", sometime "database", or just "Neo4j". It does not feel like the difference are alway deliberate. Maybe it is possible to have a consistent term for the thing the returns the error. I do not have a immediate proposal, unfortunately.

Reply:
Neo4j and Neo4j DBMS are the same thing. Neo4j DBMS or Neo4j can comprise one or more servers, which contain databases.

Reply:
I'll remove DBMS and leave only Neo4j.